### PR TITLE
sys-fs/cloudfuse:  first ebuild at 1.0

### PR DESCRIPTION
**Pull request by erikmack**

### CODE REVIEW

File: sys-fs/cloudfuse/cloudfuse-1.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,22 @@
+# Copyright 1999-2015 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+
+EAPI=5
+inherit eutils
```
Comment (alunduil) [inline sys-fs/cloudfuse/cloudfuse-1.0.ebuild]: I prefer putting newlines before and after the inherit lines.  Just personal preference and style not a requirement.  Your choice on whether you like that style.

File: sys-fs/cloudfuse/cloudfuse-1.0.ebuild [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+DESCRIPTION="A FUSE filesystem for Rackspace's Cloud Files"
+HOMEPAGE="http://redbo.github.io/cloudfuse/"
+SRC_URI="https://github.com/redbo/cloudfuse/archive/${PV}.tar.gz"
```
Comment (alunduil) [inline sys-fs/cloudfuse/cloudfuse-1.0.ebuild]: Should be ``SRC_URI="https://github.com/redbo/${PN}/archive/${PV}.tar.gz"``

File: sys-fs/cloudfuse/cloudfuse-1.0.ebuild [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+SLOT="0"
+KEYWORDS="~amd64"
+IUSE=""
+
+DEPEND="
+	net-misc/curl
+	sys-fs/fuse
+	dev-libs/openssl:0
+	dev-libs/libxml2
+"
```
Comment (alunduil) [inline sys-fs/cloudfuse/cloudfuse-1.0.ebuild]: I sort these but again that's style you can choose to adopt or not.

**Comment by alunduil** [comment]
Looking real good.  Just a couple of minor things and this is ready for merge.

**Comment by erikmack** [comment]
It's ready for another look, thanks.

**Comment by alunduil** [comment]
Looks good to me.  I'll merge and add to tree on Saturday.

**Pull request merged**